libs/sdk-ui/src/base/localization/TranslationsCustomizationProvider/index.ts

```diff
@@ -12,5 +12,5 @@ export {
     pickCorrectInsightWording,
```

Comment:
Please don't forget to update the copyright accordingly.

Reply:
fixed